### PR TITLE
Added terminology and wording section

### PR DESCRIPTION
**Pull request by cshinn**
## Description
This PR creates a document for the terminology and wording page and includes the changes that were previously in truncation.md

## Changes

- Deletes truncation.md
- Adds terminology-and-wording.md

### CODE REVIEW

**Comment by LHinson** [comment]
Great @cshinn! Given how long this thing is, can we add an index up top so people can easily find things instead of scrolling? Other than that, looks good!

File: styles/terminology-and-wording/terminology-and-wording.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+### Additional truncation guidelines.
+
+  1. Avoid abbreviations or truncated text in navigation items (first, second, and third levels of navigation in the masthead; left navigation).
+  2. In any container holding a string, if there is not sufficient room for the full spelling or hyphenation of a word, abbreviate the text according to the English abbreviation rules [actually, should be according to our rules too].  Do not truncate text in column headings.
+  3. Indicate truncated text with an ellipsis (…).  If the text is part of a link, the ellipsis should be part of the link as well.
+  4. Leave no fewer than 4 characters when truncating text, and preferably leave enough characters to give a fair idea of what the string says.  For example, don't truncate demo1.internal-el6.satellite to de...
+  5. Ensure that there is at least one method for the user to view the entire string.  We recommend the use of a [toolip](https://www.patternfly.org/pattern-library/widgets/#tooltip) (useful for less than 150 characters or so), but other options include: expanding rows and overlays.
```
Comment (serenamarie125) [inline styles/terminology-and-wording/terminology-and-wording.md]: typo here "toolip"

File: styles/terminology-and-wording/terminology-and-wording.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+| Group headings, group boxes                                | Headline       |
+| List boxes: Introductory text                              | Sentence       |
+| List boxes: List box labels                                | Headline       |
+| Prompts                                                    | Sentence       |
+| Radio buttons labels                                       | Sentence       |
+| States (as shown in tables, filters, e. g. Up and Running) | Headline       |
+| Status messages                                            | Sentence       |
+| Tab labels                                                 | Headline       |
+| Text box labels                                            | Headline       |
+| Tool tips                                                  | Sentence       |
```
Comment (serenamarie125) [inline styles/terminology-and-wording/terminology-and-wording.md]: Can we change this to "tooltip" ?  In the Pattern Library, it is referenced as a single word

File: styles/terminology-and-wording/terminology-and-wording.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+| Percent | In text, use the word percent. For example: 10 percent to 20 percent. In tables, use the percent symbol. For example: 10% to 20% |
+| Question mark	| Place a question mark at the end of a question. For example:Are you sure you want to delete this file? |
```
Comment (serenamarie125) [inline styles/terminology-and-wording/terminology-and-wording.md]: Add a space after "For example:"

**Comment by LHinson** [comment]
and this one? @cshinn

**Comment by cshinn** [comment]
@LHinson @serenamarie125 I've updated the page based on your comments

**Comment by LHinson** [comment]
LGTM. Thanks @cshinn!

**Comment by LHinson** [comment]
@serenamarie125 @mcarrano please review

File: styles/terminology-and-wording/terminology-and-wording.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+### More Information Link
+
+As a user, I want to see a single link that directs me to a page or document with additional information to learn more about a particular subject or task. As the user is reading the sentence, they have the option to click on the documentation or page link to view the documentation. This documentation could live within the same site or outside of the site. If it lives outside the site, we should let the user know by adding an icon to make this clear.
+
+### General Rules
+- The link should be included in a complete sentence, use punctuation and sentence style capitalization.
+- Names or titles of a document or page should use header style capitalization.
+- The name of the document should should use link text styling to indicate that it is clickable. Check out an example of the link text under Styles Samples on the Typography page.
+- Some recommendations for wording include:
+  - View <Document Link> for more information.
```
Comment (mcarrano) [inline styles/terminology-and-wording/terminology-and-wording.md]: The text <Document Link> is not showing up when I view the file.

**Comment by LHinson** [comment]
@cshinn just getting this back on your radar since it's been sitting here awhile.

**Comment by cshinn** [comment]
@LHinson @mcarrano PR has been updated based on feedback

**Comment by mcarrano** [comment]
LGTM.  @LHinson if you are satisfied with the changes, please go ahead and merge.

**Pull request merged**